tests/tests.js

```diff
@@ -249,23 +249,31 @@ describe('Get Zone letter from lat', function(){
     });
   });
 });
-describe('Parse USNG', function(){
-  describe('with single digit zone', function(){
-    it('should return zone=5; letter=Q', function(){
+describe('Parse USNG', function() {
+  describe('with single digit zone', function () {
+    it('should return zone=5; letter=Q', function () {
       var parts = {};
       converter.parseUSNG_str("5Q", parts);
       chai.assert.equal(5, parts.zone);
       chai.assert.equal('Q', parts.let);
     });
   });
-  describe('with two digit zone', function(){
-    it('should return zone=12; letter=S', function(){
+  describe('with two digit zone', function () {
+    it('should return zone=12; letter=S', function () {
       var parts = {};
       converter.parseUSNG_str("12S", parts);
       chai.assert.equal(12, parts.zone);
       chai.assert.equal('S', parts.let);
     });
   });
+  describe('with outside of valid range', function () {
+    it('should error when zone is 0', function () {
```

Comment:
The word "error" should be replaced with "err"; "error" is a noun while "err" is a verb meaning 'to make an error'.